src/Makefile

```diff
@@ -364,11 +368,14 @@ ifeq ($(COMP),gcc)
 	CXX=g++
 	CXXFLAGS += -pedantic -Wextra -Wshadow
 
-	ifeq ($(arch),$(filter $(arch),armv7 armv8))
+	ifeq ($(arch),$(filter $(arch),armv7 armv8 riscv64))
 		ifeq ($(OS),Android)
 			CXXFLAGS += -m$(bits)
 			LDFLAGS += -m$(bits)
 		endif
+		ifeq ($(ARCH),riscv64)
+                        CXXFLAGS += -latomic
+                endif
```

Comment:
spaces vs tabs here?

Reply:
Currently the file already has some weird inconsistencies switching from tabs to spaces
https://github.com/official-stockfish/Stockfish/blob/79c5f3a69247b44ee6362a3d9236cd9bc048c5f5/src/Makefile#L381-L383

Reply:
My bad, updated the commit with tabs